inspirehep/demosite/data/collection.json

```diff
@@ -0,0 +1,82 @@
+[
```

Comment:
This file is not really demo, but should actually be considered real configuration. Is there a better place for it?